monai/data/image_reader.py

```diff
@@ -24,21 +24,25 @@
 from .utils import is_supported_format
 
 if TYPE_CHECKING:
+    import cuimage
     import itk  # type: ignore
     import nibabel as nib
+    import openslide
```

Comment:
please update the package requirements in `requirements-dev.txt`, `docs/requirements.txt`, `setup.cfg`